Nos/Assets/Localization/Localized.swift

```diff
@@ -239,8 +251,37 @@ enum Localized: String, Localizable, CaseIterable {
     case eventSource = "Raw Event"
     case loading = "Loading..."
     case viewSource = "View Source"
-    case reportEventContent = "This content has been reported for {{ report_category }} using NIP-69 vocabulary https://github.com/nostr-protocol/nips/pull/457"
+    case reportEventContent = "This content has been reported for {{ report_category }}."
     case select = "Select"
+    
+    case nip56code_CL = "Coarse Language"
```

Comment:
These codes are already localized in the `ReportCategory` type, although we don't have a good way to instantiate a `ReportCategory` from, say, the two letter code we would find in a report. Probably we need to write a static function on `ReportCategory` that takes a report `code` and recursively searches the `topLevelCategories` data for a matching category. Then you can get the localized name using the categories `displayName`.